packages/transport-native-ble/src/api/bleApi.ts

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable require-await */
+import { Device as BleDevice } from 'react-native-ble-plx';
+
+import {
+    AbstractApi,
+    AbstractApiConstructorParams,
+    DEVICE_TYPE,
+} from '@trezor/transport/src/api/abstract';
+import * as ERRORS from '@trezor/transport/src/errors';
+import { AsyncResultWithTypedError } from '@trezor/transport/src/types';
+
+import { log } from '../logs';
+import { nativeBleManager } from './nativeBleManager';
+
+interface ConstructorParams extends AbstractApiConstructorParams {}
+
+const DEBUG_LOGS = true;
+
+const debugLog = (...args: any[]) => {
```

Comment:
now it should be possible to pass logger instance to BleApi from outside. So feel free to use it if you find it useful.